dist/tools/ethos/start_network.sh

```diff
@@ -31,22 +30,38 @@ cleanup() {
 }
 
 start_uhcpd() {
+    ip route add ${PREFIX} via fe80::2 dev ${TAP}
```

Comment:
```suggestion
    ip route add "${PREFIX}" via fe80::2 dev "${TAP}"
```

I'm aware that there is no chance that there will ever be spaces, but these CI warnings sprinkled all over the code make me crazy when reading it.